test/utils-tests.jl

```diff
@@ -346,6 +346,30 @@ end
     @test hlat_out[1] ≈ -0.6845757329085267
 end
 
+# Test hor2eq
+# The values used for the testset are from running the code. However they have been
+# correlated with the output from hor2eq routine of IDL AstroLib, with
+# differences only in the least significant digits.
+@testset "hor2eq" begin
+    ra_o, dec_o, ha_o = hor2eq(43.6879, 56.684, AstroLib.J2000, B1950=true)
+    @test ra_o ≈ 259.20005705918317
+    @test dec_o ≈ 49.674706171288655
+    @test ha_o ≈ 19.40545272279752
```

Comment:
```
GDL> hor2eq, 43.6879d0, 56.684d0, 2451545d0, ra_o, dec_o, ha_o, /B1950
GDL> print, ra_o, dec_o, ha_o
       259.20239       49.674907       291.08179
```
Do you have a clue about why `ra_o` and `dec_o` differ already on the 4th-5th most significant digit?

`ha_o` is completely different: you converted to hours, but docstring says it's in degrees, but when I convert back to degrees the value of this quantity exactly matches the value in IDL AstroLib, so there is no problem with it, only the RA and Dec coordinates.